### PR TITLE
LIP23

### PR DESCRIPTION
**Pull request by kyriediculous**
I wanted to create a poll for QA But I can't because there are no polls in proposed status.

### CODE REVIEW

**Review by adamsoffer — CHANGES_REQUESTED** [review]
Can you add a dummy discussions link? It's a required field according to the template. And you can remove the  optional fields from frontmatter if they're not applicable.

**Comment by kyriediculous** [comment]
added dummy link !

File: LIPs/LIP-23.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+lip: 23
+title: Is Michael Jordan the GOAT ? 
+author: Nico Vergauwen (@kyriediculous)
+type: Informational
+status: Proposed
+created: 2020-05-05
+discussions-to: https://en.wikipedia.org/wiki/Michael_Jordan
+requires (*optional): <LIP number(s)>
```
Comment (adamsoffer) [inline LIPs/LIP-23.md]: Can you remove the optional `requires`, `replaces`, and `part-of` fields since they're not applicable?

**Review by adamsoffer — CHANGES_REQUESTED** [review]
Can you add a discussions link? It's a required field according to the template. And you can remove the  optional fields from frontmatter if they're not applicable.

File: LIPs/LIP-23.md (hunk 1 above)
Comment (kyriediculous) [reply LIPs/LIP-23.md]: done !

Comment (adamsoffer) [reply LIPs/LIP-23.md]: haha just occurred to me why you assigned the number 23. Nice touch!

**Pull request merged**